sendgrid_test.go

```diff
@@ -130,6 +130,21 @@ func TestLicenseYear(t *testing.T) {
 	}
 }
 
+func TestRepoFiles(t *testing.T) {
+	fs := []string{"Docker", "docker-compose.yml", ".env_sample", ".gitignore", ".travis.yml", ".codeclimate.yml", "CHANGELOG.md", "CODE_OF_CONDUCT.md", "CONTRIBUTING.md", ".github/ISSUE_TEMPLATE", "LICENSE.md", ".github/PULL_REQUEST_TEMPLATE", "README.md", "TROUBLESHOOTING.md", "USAGE.md", "USE_CASES.md"}
+	for _, f := range fs {
+		if _, err := os.Stat(f); os.IsNotExist(err) {
+			if strings.HasPrefix(strings.ToLower(f), "docker") {
+				if _, err := os.Stat("docker/" + f); os.IsNotExist(err) {
+					t.Errorf("Repo files do not exist: %v and %v", f, "docker/"+f)
```

Comment:
nit: string concatination is not necessary. Move `"docker/"` part to the template:
```
"<...> %v and "docker/%v", f, f)
```

Or even:
```
`"<...> %[1]v and "docker/%[1]v", f)`
```